Fix docs inconsistencies (#713)
ONSAS/ONSAS: docs/src/creatingModels.md

Each struct has its own _fields_ with specific names, used to store each corresponding property or information. Each field is obtained or assigned using _structName.fieldName_. A description of each struct and its fields follows at next. It is highly recommended to read the current sectiong following one of the examples presented in the documentation. 

## `materials`

> The materials vector of structs contains the information of the material behavior considered for each element.

Reviewer: ```suggestion
The materials vector of structs contains at each entry $i$ the information of the $i$-th material used in the `mesh.conecCell`.
```

 * `'elastic-rotLogStr'`: used for 1D elements (truss) under large displacements.

 * `'plastic'`: an ElastoPlastic material with isotropic hardening given by the von mises flow rule for the plane strain element. The parameters are introduced as: $p_1=E$ , $p_2 = K$ and $p_3=\sigma_{Y,0}$.

 * `'plastic-rotEngStr'`: an ElastoPlastic material .

 * `'plastic-rotLogStr'`: an ElastoPlastic material .

### `materials(i).modelParams`

A vector with the properties of this material model.
```math
[ p_1 \dots p_{n_P} ]
```
where $n_P$ is the number of parameters of the constitutive model and $\mathbf{p}$ is the vector of constitutive parameters.

### `materials(i).density`

This is the scalar value of the density of the material.

### `materials(i).nodalMass`

This fields sets a vector of nodal masses components $[m_x, m_y, m_z]$ that is assigned to nodes.

## `elements`

> The element vector of structs contains the information about the type of finite elements used and their corresponding parameters.

Author: ```suggestion
The elements vector of structs contains at each entry $i$ the information of the $i$-th finte element type used in the `mesh.conecCell` for each element.
```

> The elements vector of structs contains at each entry $i$ the information of the $i$-th finte element type used in the `mesh.conecCell` for each element.

Reviewer: ```suggestion
The elements vector of structs contains at each entry $i$ the information of the $i$-th finite element type used in the `mesh.conecCell` for each element of the mesh.
```

### `elements(i).elemType`

> A string with the names of the elements used: `node`, `truss`, `frame`, `triangle` or `tetrahedron`. Other auxiliar types such as `edge` are also available

Reviewer: ```suggestion
A string with the type of element, being: `node`, `truss`, `frame`, `edge`, `triangle` or `tetrahedron`.
```

### `elements(i).elemTypeParams`

> A scalar or vector with auxiliar params information, required for some element types:

Reviewer: ```suggestion
A scalar or vector with auxiliary parameters, required for some element types:
```

### `materials.modelParams`

> A vector with the material properties of each material used in the model. The $i$-th entry of the vector, contains:

Reviewer: Thanks Mauricio i would say
```suggestion
A vector with the properties of this material model.
```
The ith should be moved since now materials is a vector of structs... There might be necessary to change the header to materials(i).modelParams... And that for elements and boundaryconds too

```math
[ p_1 \dots p_{n_P} ]
```
where $n_P$ is the number of parameters of the constitutive model and $\mathbf{p}$ is the vector of constitutive parameters.

### `material.density`

> This is a value of the density considered in the material model.

Reviewer: ```suggestion
This is the scalar value of the density of the material.
```

See the `crossSectionProps.m` function for more details.

#### 2D elements

For 2D elements such as `triangle` in this field a float number representing the thickness of the element is set.   

### `elements(i).aeroCoefFunctions`
If a frame aerodynamic analysis is desired, the drag, lift and pitch moment functions should be defined using this field. This field should contain a cell with either the strings of the functions or the definition of anonymous functions for draf lif and pitch moment in that order. Each function must receive as first input the incidence angle and as second the Reynolds number. For some `elemCrossSecParams` like `'circle'` internal built-in functions are set as default thus there is no need to set this field.

### `elements(i).chordVector`
A vector with the three coordinates of the aerodynamic chord vector (the system of coordinates considered for this is the local reference system at the undeformed configuration)

## `boundaryConds`

> The materials vector of structs contains the information of the boundary conditions of the finite element model.

Author: ```suggestion
The boundary condtion vector of structs contains at each entry $i$ the information of the $i$-th boundary condtion used in the `mesh.conecCell` for each element.
```

### `boundaryConds(i).loadsTimeFact`
cell with the inline function definitions of load factors of the loads applied of an empty array.

### `boundaryConds(i).loadsBaseVals`
cell with the (row) vector of the components of the load case
```math
[ f_x,  \, m_x, \, f_y, \, m_y, \, f_z, \, m_z ]
```
where $f_i$ are the components of forces and $m_i$ are the moments. Both forces or moments are considered per unit of length in the case of `truss`/`frame`/`edge` elements, or per unit of area in the case of `triangle`.

### `boundaryConds(i).userLoadsFileName`
string with the filename of the `.m` function file provided by the user that can be used to apply forces not given by time-varying loadFactors. This function file should be placed in the example folder and it must receive two arguments:  t (the time) and UsCell (a cell with: {the current displacement, velocity and acceleration} ). The function should one forces vector with the size of all the degrees of freedom of the problem (in global coordinates).

### `boundaryConds(i).imposDispDofs`

> vector or single local degree/s of freedom imposed (integers from 1 to 6)

Reviewer: ```suggestion
vector with the local degrees of freedom imposed (integers from 1 to 6)
```

### `boundaryConds(i).imposDispVals`

> vector/s containing the value/s of the displacements imposed.

Reviewer: ```suggestion
vector with the values of the imposed displacements at the corresponding `imposDispDofs`.
```

string with the filename of the `.m` function file provided by the user that can be used to apply forces not given by time-varying loadFactors. This function file should be placed in the example folder and it must receive two arguments:  t (the time) and UsCell (a cell with: {the current displacement, velocity and acceleration} ). The function should one forces vector with the size of all the degrees of freedom of the problem (in global coordinates).

### `boundaryConds.imposDispDofs`
vector or single local degree/s of freedom imposed (integers from 1 to 6)

### `boundaryConds.imposDispVals`

> vectors containing the value/s of the displacements imposed.

Author: ```suggestion
vector/s containing the value/s of the displacements imposed.
```
